print("docker flow running pushed with protection, errors at test")